docs/libp2p-resource-management.md

```diff
@@ -13,8 +13,10 @@ Good places to start are:
 
 ## Table of Contents
 
-- [libp2p Network Resource Manager <small>(`Swarm.ResourceMgr`)</small>](#libp2p-network-resource-manager-smallswarmresourcemgrsmall)
+- [libp2p Network Resource Manager (`Swarm.ResourceMgr`)](#libp2p-network-resource-manager-swarmresourcemgr)
```

Comment:
We don't want this in the TOC.  Lets add the comment / annotation so it isn't included when it's generated.